packages/apimonitor.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,20 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $category = 'Utilities'
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut32 = Join-Path $shortcutDir "apimonitor-x86.lnk"
+  $executablePath32 = Join-Path ${Env:ChocolateyInstall} "bin\apimonitor-x86.exe" -Resolve
```

Comment:
In case anyone is curious, these are located here due to Chocolatey's auto-shimming feature and the dependency. See for more info: https://docs.chocolatey.org/en-us/features/shim